src/viam/ur/module/ur_arm.cpp

```diff
@@ -949,10 +978,6 @@ std::string URArm::state_::state_controlled_::describe() const {
     return std::string{name()};
 }
 
-std::chrono::milliseconds URArm::state_::state_controlled_::get_timeout() const {
-    return k_noop_delay;
```

Comment:
actually, can we go ahead and delete k_noop_delay?

Reply:
Fixed, below